### ISSUE TITLE
Do not force the presence of HTML format for all books

### ISSUE THREAD

**Issue by benoit74** [openzim/gutenberg#161]
Do not force the presence of HTML format for all books anymore (see [here](https://github.com/openzim/gutenberg/blob/1007f4ae308078280c569fd83abffa8da611693e/gutenbergtozim/download.py#L120-L121)).

Especially once https://github.com/openzim/gutenberg/issues/95 has been solved, this makes little sense to always embed the HTML format, if someone only wants epub he should only get epub.

**Comment by rgaudin** [comment]
I think it was done on purpose for the following reasons:
- html is the most comfortable format to read
- html does not require an epub or pdf reader
- android browsers don't include a pdf reader as desktop browsers do
- android doesn't bundle a pdf reader
- android doesn't bundle an epub reader
- when this was done, pdf.js did not exist
- to this day, I am not aware of a usable JS epub reader we could embed.
- this scraper was built to support a large deployment of android 5 devices that are still running.

So we clearly shouldn't force HTML version(and thus remove this behavior) in the scraper and let user decide its formats but it's unlikely openZIM will create gutenberg ZIMs without html.

Maybe we should open a ticket about PDF.js. It can be handy but it brings complexity to allow both inline reading and PDF download.

**Comment by Jaifroid** [comment]
I agree HTML should be the baseline, as it's compatible with everything. Just to add to the list above, there is no native epub reader in an off-the-shelf Windows install either. Of course the user can find an app fairly easily, but it actually takes quite a long time to find one you're comfortable with for long stretches at a time.

For immediate access to the content of a book directly inside a Kiwix app on any system, there is nothing so accessible as HTML.
